lib/bolt/pal.rb

```diff
@@ -279,7 +279,7 @@ def list_tasks
     end
 
     def list_modulepath
-      @modulepath - [BOLTLIB_PATH, MODULES_PATH]
+      @original_modulepath
```

Comment:
I think we can just make this an `attr_reader`. Maybe rename the variable to `@user_modulepath` to make its meaning clear though?